tests/site.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I think we should add all roles we provide, it's skipping [on Travis](https://travis-ci.org/stackbuilders/sb-debian-base#L1932).